test/unit/specs/components/v-link.spec.js

```diff
@@ -49,6 +48,7 @@ describe('VLink', () => {
     ${'http://localhost'} | ${'A'}
   `('VLink as a $component handles click', async ({ href }) => {
     const createVLinkWrapper = (href) =>
+      // eslint-disable-next-line vue/one-component-per-file
```

Comment:
:pray:

Reply:
The annotation in the GitHub UI caused by this was bothering me. Glad we can get rid of it.